docs/UsersGuide/source/Glossary.rst

```diff
@@ -110,6 +122,9 @@ Glossary
       An interpreted programming language designed specifically for scientific data analysis and 
       visualization. Stands for NCAR Command Language. More information can be found at https://www.ncl.ucar.edu.
 
+   NDAS
+      :term:`NAM` Data Assimilation System (NDAS) data. This data is required for use of the METplus verification suite within the SRW App. The most recent 1-2 days worth of data are publicly available in PrepBufr format and can be accessed `here <ftp://ftpprd.ncep.noaa.gov/pub/data/nccf/com/rap/prod>`__. The most recent 8 days of data can be accessed `here <https://nomads.ncep.noaa.gov/pub/data/nccf/com/nam/prod/>`__.
```

Comment:
It is required if sfc and upper-air verification is desired - maybe qualify that?